openxr/src/session.rs

```diff
@@ -473,4 +495,83 @@ impl FrameWaiter {
             should_render: out.should_render.into(),
         })
     }
+
+    /// Same as .wait() but also returns whether each secondary view is active
+    /// (`XR_MSFT_secondary_view_configuration` must be loaded)
+    #[inline]
+    pub fn wait_secondary_multiple(
+        &mut self,
+        count: u32,
+    ) -> Result<(FrameState, Vec<SecondaryViewState>)> {
+        let mut vec = Vec::new();
+        vec.resize(
+            count as usize,
+            sys::SecondaryViewConfigurationStateMSFT::out(ptr::null_mut()),
+        );
+        let out = unsafe {
+            let mut secondary = sys::SecondaryViewConfigurationFrameStateMSFT::out(ptr::null_mut());
+            (*secondary.as_mut_ptr()).view_configuration_count = count;
+            (*secondary.as_mut_ptr()).view_configuration_states = vec.as_mut_ptr() as *mut _;
+            let mut x = sys::FrameState::out(&mut secondary as *mut _ as *mut _);
+            cvt((self.session.instance.fp().wait_frame)(
+                self.session.handle,
+                ptr::null(),
+                x.as_mut_ptr(),
+            ))?;
+            x.assume_init()
+        };
+        let secondary = vec
+            .into_iter()
+            .map(|x| {
+                let x = unsafe { x.assume_init() };
+                SecondaryViewState {
+                    ty: x.view_configuration_type,
+                    active: x.active.into(),
+                }
+            })
+            .collect();
+        Ok((
+            FrameState {
+                predicted_display_time: out.predicted_display_time,
+                predicted_display_period: out.predicted_display_period,
+                should_render: out.should_render.into(),
+            },
+            secondary,
+        ))
+    }
+
+    /// Same as .wait() but also returns whether the secondary view is active,
+    /// if `XR_MSFT_secondary_view_configuration` is loaded and the session has been
+    /// initialized with `begin_secondary`
+    #[inline]
+    pub fn wait_secondary(&mut self) -> Result<(FrameState, SecondaryViewState)> {
```

Comment:
Seems prudent to document the requirements on `count` for both of these.

Reply:
what `count`?

Reply:
oh, on the other one